core/urls.py

```diff
@@ -5,5 +5,5 @@
 app_name = "core"
 urlpatterns = [
     path("", views.index, name="index"),
-    path("profile/", views.profile, name="profile"),
+    path("panel/profile/<str:name>", views.profile, name="profile"),
```

Comment:
why?

Reply:
1. It should be in panel
2. What if you want to view profile of other person?

Reply:
Oh so it's intended to be able to view other's profiles